### ISSUE TITLE
Cutscene 2 tumbleweed moves weirdly

### ISSUE THREAD

**Issue by josephdong1000** [hackclub/sinerider#558]
### Description

The cutscene 2 tumbleweed doesn't actually tumble/bounce. It looked a little weird sliding on the ground smoothly when I played it on Friday. Theoretically not a very difficult change.

**Comment by polytroper** [comment]
Yeah, the plan was to make it roll but it didn’t make it into launch. We should fix this
